### PR TITLE
Feature/maps key from env

### PR DESCRIPTION
**Pull request by carmelp16**
Fixes [this issue](https://github.com/hasadna/anyway/issues/1819).

- Added the maps api key to the `.env` file.
- Added corresponding environment variable inside `docker-compose.yml`.
- Retrieve its value from env var MAPS_API_KEY and inject to the map's javascript using flask `context`.
- default value if env var is not set is the existing value used in production

### CODE REVIEW

**Comment by OriHoch** [comment]
@carmelp16 please write a description for the PR - describing the changes you did, link to the related issue this is fixing etc..

**Review by OriHoch — CHANGES_REQUESTED** [review]
looks good! only 1 change is required - the MAPS_API_KEY should have a default value which is the existing value

**Comment by carmelp16** [comment]
@OriHoch Added fixes regarding your comment :)

File: anyway/flask_app.py [hunk 1]
```diff
@@ -920,7 +922,7 @@ def index(marker=None, message=None):
     context["iteritems"] = dict.items
     context["hide_search"] = True if request.values.get("hide_search") == "true" else False
     context["embedded_reports"] = get_embedded_reports()
-    context['maps_api_key'] = secrets.get('MAPS_API_KEY')
+    context['maps_api_key'] = secrets.get('MAPS_API_KEY', DEFAULT_MAPS_API_KEY);
```
Comment (OriHoch) [inline anyway/flask_app.py:925]: the secrets get functions don't support a default value - https://github.com/hasadna/anyway/blob/dev/anyway/secrets.py#L9

**Review by OriHoch — CHANGES_REQUESTED** [review]

File: anyway/flask_app.py (hunk 1 above)
Comment (OriHoch) [reply anyway/flask_app.py:925]: also, the api key is not a secret so perhaps it shouldn't be handled by the secrets module

Comment (carmelp16) [reply anyway/flask_app.py]: You are right!
It is now fixed :)

**Review by OriHoch — APPROVED** [review]
:+1:

**Pull request merged**